plexus-utils/pom.xml

```diff
@@ -28,6 +28,8 @@ under the License.
   <groupId>org.apache.maven</groupId>
   <artifactId>plexus-utils</artifactId>
   <version>4.0.0-alpha-4-SNAPSHOT</version>
+  <name>Apache Maven Plexus-Utils</name>
+  <description>Apache Maven repackaging of Plexus Utils with immutable Dom interface and its replacement implementation.</description>
```

Comment:
DOM

Reply:
no, the interface is named `Dom`, which is not the best name IMHO...

Reply:
Really? This is confusing, it thought it is an acronym referring to Document Object Model...

Reply:
The name comes from the plexus class [`Xpp3Dom`](https://github.com/codehaus-plexus/plexus-utils/blob/master/src/main/java/org/codehaus/plexus/util/xml/Xpp3Dom.java) but I removed the `Xpp3` part.  And yes, it's referring to Document Object Model, but I would not call it `DOM`. We could rename it `XmlNode` ?

Reply:
Sounds reasonable